src/client/fuse_ll.cc

```diff
@@ -1349,6 +1453,16 @@ int CephFuse::Handle::init(int argc, const char *argv[])
 
 int CephFuse::Handle::start()
 {
+  bool is_mounted = false;
+  int err = get_ceph_mount_status(mountpoint, is_mounted);
+  if (err) {
+    return err;
+  }
+
+  if (is_mounted) {
+    return EALREADY;
```

Comment:
The mount(2) manpage seems to suggest that this error ought to be EBUSY:
```
EBUSY  An attempt was made to stack a new mount directly on top of an existing mount point
              that was created in this mount namespace with the same source and target.
```
Does FUSE driver do anything with the exit code from the program?

Reply:
Changed to EBUSY.
This exit code is to print on console in case of failure.